db.migration/sql/create_table.sql

```diff
@@ -0,0 +1,25 @@
+
+	CREATE TABLE IF NOT EXISTS roles (
```

Comment:
hm, I'm a bit confused, why do you need both liquibase and create tables? Is create tables script just for documentation or would you run it somehow?